apiserver/facades/client/sshclient/facade.go

```diff
@@ -33,20 +34,26 @@ type FacadeV2 struct {
 	*Facade
 }
 
-func internalFacade(backend Backend, leadershipReader leadership.Reader, auth facade.Authorizer, callCtx context.ProviderCallContext) (*Facade, error) {
+func internalFacade(backend Backend, leadershipReader leadership.Reader, auth facade.Authorizer, controllerTag names.ControllerTag, callCtx context.ProviderCallContext) (*Facade, error) {
 	if !auth.AuthClient() {
 		return nil, apiservererrors.ErrPerm
 	}
 
-	return &Facade{backend: backend, authorizer: auth, callContext: callCtx, leadershipReader: leadershipReader}, nil
+	return &Facade{backend: backend, authorizer: auth, callContext: callCtx, controllerTag: controllerTag, leadershipReader: leadershipReader}, nil
 }
 
 func (facade *Facade) checkIsModelAdmin() error {
+	isSuperUser, err := facade.authorizer.HasPermission(permission.SuperuserAccess, facade.controllerTag)
```

Comment:
We should look to be consistent with how model tag and controller tag are passed in
model tag is attached to backend in the register() function. We should add controller tag to backend too and avoid passing in separately (stick to the one approach).

Reply:
That makes perfect sense. I've just moved the controllertag to the backend.